src/index.html

```diff
@@ -11,8 +12,67 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Browsers cannot interpret SCSS files directly. You need to compile `index.scss` into a CSS file and link to the resulting CSS file instead.